docs/reference/api/docker_remote_api_v1.18.md

```diff
@@ -1501,7 +1501,7 @@ Search for an image on [Docker Hub](https://hub.docker.com).
 
 **Example request**:
 
-    GET /images/search?term=sshd HTTP/1.1
+    GET /v1.18/images/search?term=sshd HTTP/1.1
```

Comment:
Even with static docs, it isn't too bad – you just have to do a big regex search/replace for each new version.

I did something like this: `s/(GET|POST|PATCH|PUT|HEAD|OPTIONS) /(.+?) HTTP/$1 /v1.25/$2 HTTP/`